website/src/documentation/patterns/side-inputs.md

```diff
@@ -46,29 +54,36 @@ For instance, the following code sample uses a `Map` to create a `DoFn`. The `Ma
 {% github_sample /apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/snippets/Snippets.java tag:SideInputPatternSlowUpdateGlobalWindowSnip1
 %}
 ```
+```py
+No sample present.
+```
 
 
 ## Slowly updating side input using windowing
 
-You can read side input pcollection periodically into distinct windows.
-Later, when you apply side input to your main input, windows will be matched automatically 1:1.
-This way, you can guarantee side input consistency on the duration of the single window.
+You can read side input data periodically into distinct PCollection windows.
+Later, when you apply the side input to your main input, each main input
+window is automatically matched to a single side input window.
+This guarantees consistency on the duration of the single window,
+meaning that each window on the main input will be matched to a single
+version of side input data.
 
-To do this, you can utilize PeriodicSequence PTransform that will generate infinite sequence
-of elements with some real-time period:
+Described approach can be implemented using combination of
```

Comment:
How about something like this: "To read side input data periodically... use a combination of PeriodicSequence..."

Reply:
I'm a bit confused here since adding text you suggest is pretty much re-writing same thing that is a header and previous abstract. In next iteration I tried to remove this abstract overall since list of steps provides same information.

Reply:
Gotcha - the intent is to summarize the previous abstract, so that it's easier for readers to skim the doc.